R/model.R

```diff
@@ -495,10 +558,19 @@ anc_testing_artcov_mf <- function(quarter_ids, anc_testing, naomi_mf) {
 
 #' @rdname anc_testing_prev_mf
```

Comment:
Can you update the documentation for `anc_testing_prev_mf` to have an `@param` section for `quarter_id` instead of `quarter_ids`.